app-text/texlive/texlive-2021.recipe

```diff
@@ -0,0 +1,958 @@
+# braces, as described above.  (This used to be HOMETEXMF.)  ~ expands
+# to #USERPROFILE# on Windows, $HOME otherwise.
+#
+###		TEXMFHOME = ~/texmf
+#
+# TEXMFVAR, where texconfig/updmap/fmtutil store cached runtime data.
+#
+###		TEXMFVAR = ~/.texlive2014/texmf-var
+#
+# TEXMFCONFIG, where texconfig/updmap/fmtutil store configuration data.
+#
+###		TEXMFCONFIG = ~/.texlive2014/texmf-config
+#
+# List all the texmf trees. For an explanation of what they are, see the
+# TeX Live manual.
+#
+# For texconfig to work properly, TEXMFCONFIG and TEXMFVAR should be named
+# explicitly and before all other trees.
+#
+# TEXMFLOCAL precedes TEXMFDIST because locally-installed versions
+# should take precedence over distribution files -- although it is
+# generally a source of confusion to have different versions of a
+# package installed, whatever the trees, so try to avoid it.
+#
+###		TEXMF = {$TEXMFCONFIG,$TEXMFVAR,$TEXMFHOME,!!$TEXMFSYSCONFIG,!!$TEXMFSYSVAR,!!$TEXMFLOCAL,!!$TEXMFDIST}
+#
+# Where to look for ls-R files.  There need not be an ls-R in the
+# directories in this path, but if there is one, Kpathsea will use it.
+# By default, this is only the !! elements of TEXMF, so that mktexlsr
+# does not create ls-R files in the non-!! elements -- because if an
+# ls-R is present, it will be used, and the disk will not be searched.
+# This is arguably a bug in kpathsea.
+#
+###		TEXMFDBS = {!!$TEXMFSYSCONFIG,!!$TEXMFSYSVAR,!!$TEXMFLOCAL,!!$TEXMFDIST}
+#
+# The system trees.  These are the trees that are shared by all users.
+# If a tree appears in this list, the mktex* scripts will use
+# VARTEXFONTS for generated files, if the original tree isn't writable;
+# otherwise the current working directory is used.
+#
+###		SYSTEXMF = $TEXMFSYSVAR;$TEXMFLOCAL;$TEXMFDIST
+#
+# First writable tree here is used by Lua(La)TeX for the font cache.
+# LuaLaTeX uses the value here, while ConTeXt uses the same variable but
+# from texmfcnf.lua; therefore the two values should be kept in sync.
+# (As should everything else in texmf.cnf <-> texmfcnf.lua.)
+#
+###		TEXMFCACHE = $TEXMFSYSVAR;$TEXMFVAR
+#
+# Where generated fonts may be written.  This tree is used when the sources
+# were found in a system tree and either that tree wasn't writable, or the
+# varfonts feature was enabled in MT_FEATURES in mktex.cnf.
+#
+###		VARTEXFONTS = $TEXMFVAR/fonts
+#
+# On some systems, there will be a system tree which contains all the font
+# files that may be created as well as the formats.  For example
+#   TEXMFVAR = /var/lib/texmf
+# is used in many distros.  In this case, set VARTEXFONTS like this
+#VARTEXFONTS = $TEXMFVAR/fonts
+# and do not mention it in TEXMFDBS (but _do_ mention TEXMFVAR).
+#
+# Remove $VARTEXFONTS from TEXMFDBS if the VARTEXFONTS directory is below
+# one of the TEXMF directories (avoids overlapping ls-R files).
+fixKSearchPaths()
+{
+	TEXMFCNF=$sourceDir/texk/kpathsea/texmf.cnf
+	sed -i -e "s@TEXMFROOT =.*@TEXMFROOT = $dataDir/texlive@" $TEXMFCNF
```

Comment:
You can combine this lines

Reply:
I have not changed this part from the 2014 recipe. Maybe I should delete it, now that this version "works" so that one can see the differences better?

Reply:
Yep, pls delete the old recipe then.